src/basic/env-util.c

```diff
@@ -779,7 +779,7 @@ int serialize_environment(FILE *f, char **environment) {
                 if (!ce)
                         return -ENOMEM;
 
-                fprintf(f, "env=%s\n", *e);
+                fprintf(f, "env=%s\n", ce);
```

Comment:
Do you know why the following command does not work according to the shell rules
```sh
systemd-run --property 'Environment=A=A\x0aB' date
Invalid environment assignment: A=A
B
Failed to create bus message: Invalid argument
```
?